erpnext/public/js/utils.js

```diff
@@ -293,7 +293,7 @@ $.extend(erpnext.utils, {
 	},
 	copy_parent_value_in_all_row: function(doc, dt, dn, table_fieldname, fieldname, parent_fieldname) {
 		var d = locals[dt][dn];
-		if(d[fieldname]){
+		if(d[parent_fieldname]){
```

Comment:
- Since this is a utility function, all logic needs to be generic (that serves everyone). 
- `parent_fieldname` and `fieldname` are both critical to this function.
- The check is to make sure that someone does not accidentally pass a non-existent fieldname, mistyped fieldname or no fieldname at all. So you could check for both these fields since they are required.
```suggestion
		if(d[parent_fieldname] && d[fieldname]){
```